Q/Qt/build_tarballs.jl

```diff
@@ -14,6 +14,7 @@ sources = [
 ]
 
 script = raw"""
+apk del ninja
```

Comment:
@giordano this worries me -- will we now have to adjust all kinds of packages to `apkg del ninja` or `apk add` ninja or samurai? Perhaps a tweak in BB or BBB or so is necessary to take care of this change (?!) is necessary?

Reply:
I'd like to understand what's wrong with Alpine.  Honestly, that a package becomes broken seems a bit worrying to me.

I think one should try to rebuild the RootFS and see if that solves the issue